docs/articles/documentation/reference/test-api/requesthook/README.md

```diff
@@ -103,3 +103,35 @@ async onResponse (event) {
     throw new Error('Not implemented');
 }
 ```
+
+{% include notes-warnings/requesthook-onresponse-no-stub-headers.md href='#stub-response-headers' %}
+
+### Stub Response Headers
+
+Before it passes an incoming response to the application, `RequestHook` can change the response headers.
+
+You can use the `RequestHook._onConfigureResponse()` method to substitute response headers, effectively mocking part of the response.
+
+> Important! The `RequestHook._onConfigureResponse()` method is private. Exercise caution when you implement this method in your request hooks.
```

Comment:
why `implement`? it is my understanding that this method works out of the box...
also, `exercise caution` is unnecessarily formal and old-fashioned
```suggestion
> Important! The `RequestHook._onConfigureResponse()` method is private. Be cautious when you add this method to your request hooks.
```